includes/Classifai/Features/ImageCropping.php

```diff
@@ -163,6 +164,28 @@ public function generate_smart_crops( array $metadata, int $attachment_id ): arr
 		return $metadata;
 	}
 
+	/**
+	 * Adjust the media library attachment size to account for cropped thumbnails.
+	 *
+	 * @param array $response Array of prepared attachment data.
+	 * @return array
+	 */
+	public function set_media_library_attachment_size( array $response ): array {
```

Comment:
Because this is hooked to a core WordPress filter, someone else may filter this and return invalid data, which would then cause these type checks to fail. My general approach is if we have a function hooked to a core WordPress filter, we don't add these type hints
```suggestion
	public function set_media_library_attachment_size( $response ) {
```

Reply:
Handled in https://github.com/10up/classifai/pull/971/commits/a07975d00db40d02a3106dc3ec7e3a9e00fbd4aa